package.json

```diff
@@ -55,7 +55,7 @@
     "webpack": "^1.13.0"
   },
   "peerDependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1"
+    "react": ">=15.0.1",
```

Comment:
From what I understand this would also allow React at version 24 which is problematic because right now we can hardly tell if it still works with that version.

Most people will therefor specify cross-major version ranges like this:

```
"react": "^15.0.1 || ^16.0.0 || ^17.0.0 || ^18.0.0"
```

which is a bit tedious but prevents frustration on breaking updates.